vizro-core/tests/unit/vizro/models/_navigation/test_navigation.py

```diff
@@ -49,52 +49,13 @@ def test_field_invalid_pages_input(self, pages, expected_error):
 class TestNavigationBuild:
     """Tests navigation build method."""
 
-    def test_navigation_default(self, accordion_from_page_as_list):
-        result_navigation = vm.Navigation()
-        result_navigation.pre_build()
-
-        # setting accordion id to fix the random id generation
-        result_navigation._selector.id = "accordion_list"
-
-        result = json.loads(json.dumps(result_navigation.build(), cls=plotly.utils.PlotlyJSONEncoder))
-        expected = json.loads(json.dumps(accordion_from_page_as_list, cls=plotly.utils.PlotlyJSONEncoder))
-
-        assert result == expected
-
-    @pytest.mark.parametrize("navigation_pages", [(["Page 1", "Page 2"]), None])
-    def test_navigation_same_result_with_different_config(self, navigation_pages, accordion_from_page_as_list):
-        result_navigation = vm.Navigation(pages=navigation_pages)
-        result_navigation.pre_build()
-
-        # setting accordion id to fix the random id generation
-        result_navigation._selector.id = "accordion_list"
-        accordion_from_page_as_list.children.id = "accordion_list"
-
-        result = json.loads(json.dumps(result_navigation.build(), cls=plotly.utils.PlotlyJSONEncoder))
-        expected = json.loads(json.dumps(accordion_from_page_as_list, cls=plotly.utils.PlotlyJSONEncoder))
-
-        assert result == expected
-
-    def test_navigation_pages_as_list(self, pages_as_list, accordion_from_page_as_list):
-        result_navigation = vm.Navigation(pages=pages_as_list)
-        result_navigation.pre_build()
-
-        # setting accordion id to fix the random id generation
-        result_navigation._selector.id = "accordion_list"
-
-        result = json.loads(json.dumps(result_navigation.build(), cls=plotly.utils.PlotlyJSONEncoder))
-        expected = json.loads(json.dumps(accordion_from_page_as_list, cls=plotly.utils.PlotlyJSONEncoder))
-
-        assert result == expected
-
-    def test_navigation_pages_as_dict(self, pages_as_dict, accordion_from_pages_as_dict):
-        result_navigation = vm.Navigation(pages=pages_as_dict)
-        result_navigation.pre_build()
-
-        # setting accordion id to fix the random id generation
-        result_navigation._selector.id = "accordion_dict"
-
-        result = json.loads(json.dumps(result_navigation.build(), cls=plotly.utils.PlotlyJSONEncoder))
-        expected = json.loads(json.dumps(accordion_from_pages_as_dict, cls=plotly.utils.PlotlyJSONEncoder))
-
+    @pytest.mark.parametrize("pages", [["Page 1", "Page 2"], {"Page 1": ["Page 1"], "Page 2": ["Page 2"]}, None])
```

Comment:
@nadijagraca I've reviewed the tests again and removed all of these here, as we already tested all of these in the `test_accordion.py`. So, these are essentially duplicates. 

The only test we need is that the `accordion.build()` returns the same as the `navigation.build()`, similar to how we have it for our tests in `test_filter.py`. The only test that would be worth adding here is if, at some point, we make the Accordion public. In this case, we should test for the use case of precedence, so if pages are provided to Navigation and Accordion, the configuration for Accordion will have precedence. 

However, given that this is private for now, this is not required yet.